### PR TITLE
(4.0.0 branch) Updating the default ports documentation

### PR DESCRIPTION
**Pull request by nilminiwso2**
- Adding streaming integration ports
- Consolidating the link between 'Changing default ports' and the 'list of default ports' for all of the API Manager runtimes.
- Moved the section on 'disabling the HTTP transport for the API-M runtime' from the Ports documentation to https://apim.docs.wso2.com/en/latest/install-and-setup/setup/security/configuring-transport-level-security/

Fixes https://github.com/wso2/docs-apim/issues/3956

### CODE REVIEW

File: en/docs/install-and-setup/setup/security/configuring-transport-level-security.md [hunk 1]
```diff
@@ -82,7 +83,42 @@ By default, all WSO2 products pass "WSO2 Carbon Server" as the server value in H
...
+    It is recommended to disable the HTTP transport in an API Manager production setup. Using the `Bearer` token over HTTP is a violation of the OAuth specification and can lead to security vulnerabilities.
+
+API Manager has two HTTP transports.
+
+-   Passthru (API Traffic) Transport
+-   Servlet (UI Traffic and Admin service access) Transport
+
+See the instructions given below to disable these transports.
+
+### Disabling Passthrough Transport
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/setup/security/configuring-transport-level-security.md:98]: @tharindu1st 
Could you confirm if this is accurate? I can see the passthrough config section in the Config Catalog[1], however, it conflicts with the toml headers we have here in this doc.

[1] https://apim.docs.wso2.com/en/latest/reference/config-catalog/#https-pass-through-transport-non-blocking-mode

File: en/docs/install-and-setup/setup/security/configuring-transport-level-security.md [hunk 2]
```diff
@@ -82,7 +83,42 @@ By default, all WSO2 products pass "WSO2 Carbon Server" as the server value in H
...
+
+Add the following configuration in the `deployment.toml` file (stored in the `<API-M_HOME>/repository/conf` folder).
+
+```toml
+[transport.passthru_http.listener]
+enable = false
+```
+
+### Disabling Servlet Transport
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/setup/security/configuring-transport-level-security.md:107]: @tharindu1st I was unable to find the toml configs for this.
According to the config catalog, the the toml configs for the servlet transport is [1]. However, there is no parameter for disabling. Is this disabled by default?

[1] https://apim.docs.wso2.com/en/latest/reference/config-catalog/#http-servlet-transport

File: en/docs/install-and-setup/setup/deployment-best-practices/changing-the-default-ports-with-offset.md [hunk 3]
```diff
@@ -145,3 +148,49 @@ listenerConfigurations:
...
+    # value to uniquely identify a server
+id: wso2-si
+    # server name
+name: WSO2 Streaming Integrator
+    # server type
+type: wso2-si
+    # ports used by this server
+ports:
+    # port offset
+    offset: 1
```
Comment (dnwick) [inline en/docs/install-and-setup/setup/deployment-best-practices/changing-the-default-ports-with-offset.md:173]: For SI tooling we have default port offset 3 and for SI we have it as 0

Comment (nilminiwso2) [reply en/docs/install-and-setup/setup/deployment-best-practices/changing-the-default-ports-with-offset.md]: updated

File: en/docs/install-and-setup/setup/security/configuring-transport-level-security.md [hunk 4]
```diff
@@ -82,7 +83,42 @@ By default, all WSO2 products pass "WSO2 Carbon Server" as the server value in H
     server="WSO2 Carbon Server"
     ```
 
-!!! info
-    See the [Security Guidelines for Production Deployment](https://docs.wso2.com/display/ADMIN44x/Security+Guidelines+for+Production+Deployment) for the full list of security-related recommendations for WSO2 products.
+## Disabling HTTP Transports
+
+!!! Note
```
[the remark this answers is not in the document]
Comment (tharindu1st) [reply en/docs/install-and-setup/setup/security/configuring-transport-level-security.md:88]: yes nothing to add thn that

File: en/docs/install-and-setup/setup/security/configuring-transport-level-security.md (hunk 1 above)
Comment (tharindu1st) [reply en/docs/install-and-setup/setup/security/configuring-transport-level-security.md:98]: default value is true

File: en/docs/install-and-setup/setup/security/configuring-transport-level-security.md (hunk 2 above)
Comment (tharindu1st) [reply en/docs/install-and-setup/setup/security/configuring-transport-level-security.md:107]: This was needs to address from WUM update.

Comment (tharindu1st) [reply en/docs/install-and-setup/setup/security/configuring-transport-level-security.md:107]: https://github.com/wso2/product-apim/issues/11177

File: en/docs/install-and-setup/setup/security/configuring-transport-level-security.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (nilminiwso2) [reply en/docs/install-and-setup/setup/security/configuring-transport-level-security.md:88]: ack.

File: en/docs/install-and-setup/setup/security/configuring-transport-level-security.md (hunk 2 above)
Comment (nilminiwso2) [reply en/docs/install-and-setup/setup/security/configuring-transport-level-security.md:107]: ack. Will leave the doc as it is until the bug is fixed.

File: en/docs/install-and-setup/setup/security/configuring-transport-level-security.md (hunk 1 above)
Comment (nilminiwso2) [reply en/docs/install-and-setup/setup/security/configuring-transport-level-security.md:98]: That means, to disable we have to set this value to 'false' as given in the doc.
I take it that the doc is correct in that case.
Ack.

**Review by tharindu1st — APPROVED** [review]

**Pull request merged**